src/resources/SearchUsageMetrics/SearchHubs/SearchHubs.ts

```diff
@@ -0,0 +1,54 @@
+import API from '../../../APICore';
+import Resource from '../../Resource';
+import {
+    SearchHubsList,
+    SearchHubNameParams,
+    RestSearchHub,
+    SearchHubModel,
+    UpdateSearchHubParams,
+    UpdateSearchHubBucketParams,
+} from './SearchHubsInterface';
+
+export default class SearchHubs extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/searchusagemetrics/hubs`;
+
+    list() {
+        return this.api.get<SearchHubsList>(
```

Comment:
I think this should be better

```suggestion
        return this.api.get<{hubs: RestSearchHub[]}>(
```